src/store/user/reducer.js

```diff
@@ -0,0 +1,21 @@
+import { LOGIN_USER, LOGOUT_USER } from './actions'
+
+const reducer = (state = {}, action) => {
+  switch (action.type) {
+    case LOGIN_USER:
+      return {
+        ...state,
+        ...action.payload,
+      }
+    case LOGOUT_USER: {
+      // const nextState = Object.assign({}, state)
+      // delete nextState['user']
+      // return nextState
+      return state
```

Comment:
What do you plan to store in `user` state? Depending on that, you will want to clear the state when you log out, so you probably don't want to return just the current state 🙂